tools/repo/notify.py

```diff
@@ -31,6 +31,7 @@
 CALENDAR = "https://calendar.google.com/calendar/ical/d6glc0l5rc3v235q9l2j29dgovh3dn48%40import.calendar.google.com/public/basic.ics"
 
 ISSUE_LINK = "https://github.com/envoyproxy/envoy/issues?q=is%3Aissue+is%3Aopen+label%3Atriage"
+PR_LINK = "https://github.com/envoyproxy/envoy/pulls?q=is%3Apr+is%3Aopen+no%3Aassignee+draft%3Afalse+-author%3Aapp%2Fdependabot"
```

Comment:
I am wary of this addition

One of the reasons we added the script in the first place is github didn't do a good job of telling us what PRs had maintainer assigners.  If an API change auto-assigns mark roth, it won't show up in this list. 

If we want to add a link for folks to do best effort triage it's not a bad idea, but I'd be inclined to explicitly call out it's best effort, or differentiate PRs from no maintainer assigned (list below) from PRs with _no one_ assigned (your link)
/wait

Reply:
hmm, yeah - good point

what might actually be easier than it sounds is to link to add the correct filter - ie no maintainer assigned - but it would make the link quite long as it would have all the maintainers excluded

Reply:
yeah generally when I want to do a second pass I just look at the envoy PR list - it's pretty easy to see what's unassigned for the new PRs and there's never been enough of them I felt the need to filter.

Reply:
i always filter - theres usually a stack of WIP prs, and residual dependabot noise

Reply:
so - my plan didnt work - you cant filter on multiple `assignees`

the other idea i had was to automatically add an `unassigned` label to new PRs and then whenever un/assignment happens we could add/remove the label as required